src/alfasim_sdk/testing/fixtures.py

```diff
@@ -120,17 +120,19 @@ def run(self, *, number_of_threads: int = -1) -> Results:
         self.alfacase_data_folder.mkdir()
         self.alfacase_file.write_text(alfacase, encoding="utf-8")
 
-        self._run_simulator(
-            self.get_runner()
-            + [
-                "--run-simulation",
-                f"--alfacase-file={str(self.alfacase_file)}",
-                f"--number-of-threads={number_of_threads}",
-            ],
-            cwd=project_folder,
-        )
+        try:
+            self._run_simulator(
```

Comment:
You changed this function to return an error code, shouldn't this error code be checked here?

Reply:
I will not return the error code, just let subprocess do the checking.